crates/svm-runtime/src/vmcalls/storage.rs

```diff
@@ -5,11 +5,95 @@ use wasmer_runtime::Ctx as WasmerCtx;
 
 use svm_layout::VarId;
 
+macro_rules! store_n_impl {
+    ($nbytes:expr, $ctx:ident, $mem_idx:expr, $mem_ptr:expr, $var_id:expr) => {{
+        use crate::helpers;
+        use svm_layout::VarId;
+
+        let mem_ptr = $mem_ptr as usize;
+        let view = &$ctx.memory($mem_idx).view::<u8>()[mem_ptr..(mem_ptr + $nbytes)];
```

Comment:
> You presented it as if `store160` is a solution to the bytes order mismatch. If so, why not to comply with Wasm Memory byte order, and having `store160` just as a simpler API in some use-cases?

I don't want to change the original raw data. If I have got an `Address` laid out in Memory
I want these 20 bytes to stay as the raw input. It has nothing with endianness.

Reply:
Regarding checking `mem_ptr`:
If `mem_ptr` is not within the bounds of the memory - I want the program to panic.

Reply:
> If mem_ptr is not within the bounds of the memory - I want the program to panic.

It will panic anyway but the error message might not be descriptive, and it still leaves doubt regarding whether this case is expected/possible or not.